### DISCUSSION TITLE
FS19 Consumption of resources (seeds, fertilizer, lime, etc.) with Courseplay

### DISCUSSION THREAD

**Discussion by Netj0hn** [Courseplay/courseplay#6764] [Q&A] [answered]
Hello and, first of all, thanks for this great mod 🙂

I have a quick question for simulation enthusiasts of the community : in Farming Simulator 19, could any of you tell me if it is possible to make employees consume resources when actionned via Courseplay?

Indeed, I noticed that when a tasks consuming seeds, fertilizer, herbicide or lime (I may forget other ones), the vehicle reserve remains fixed.

Is this normal and expected operation, or did I forget to activate an hidden option?

Thank you in advance 🙂

Best regards,

**Comment by pvaiko** [comment]
(Guess not many have discovered yet this discussion section here)

Courseplay drivers should consume resources just like the game helpers, are you sure you have not turned on the buy fertilizer/seed/etc. options in the settings?

**Comment by Netj0hn** [comment, marked as answer]
Thanks a lot guys for the help, my buying options were disabled, but after I enabled and disabled them again, it all went fine :)

Thanks again !

Best regards,
